atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsGetCandidateCellsForSweepingBenchmarks.java

```diff
@@ -71,19 +74,34 @@ public Object fullTableScanDirtyThorough(ConsecutiveNarrowTable.DirtyNarrowTable
         return fullTableScan(table, true);
     }
 
+    @Benchmark
+    @Threads(1)
+    @Warmup(time = 20, timeUnit = TimeUnit.SECONDS)
+    @Measurement(time = 160, timeUnit = TimeUnit.SECONDS)
+    public Object fullTableScanOneWideRowThorough(VeryWideRowTable table) {
+        return fullTableScan(table.getTableRef(), table.getKvs(), table.getNumCols(), true);
+    }
+
     private int fullTableScan(ConsecutiveNarrowTable table, boolean thorough) {
+        return fullTableScan(table.getTableRef(), table.getKvs(), table.getNumRows(), thorough);
```

Comment:
It's a shame you have to unpack this, when instead there could be a `BenchmarkingTable` base class that can go to `fullTableScan`. Definitely outside scope, but I'd like to see a `TODO` here.

Reply:
added a TODO